tesler-doc-db/src/main/resources/db/migration/liquibase/data/latest/widgets/navigation/navigation-tabs.md

```diff
@@ -1,2 +1,72 @@
 # Tabs
+Tabs are needed to navigate at screen level between different views. They have their own hierarchy: first-level menu tabs are seen always on the current screen, second-level tabs are seen when you choose a tab from first-level menu, etc.
 
+For example, on the current page you can see first-level menu (*"Navigation"*, *"Widgets"*, *"Fields"*) and current second-level menu (*"Overview"*, *"Screen"*, ...). If you choose another first-level tab, you will see another set of second-level tabs. 
+
+More info about level-menu hierarchy you can find at *[Major Concepts](#/screen/getting-started/view/screen/)*
+
+## How to create?
+Tabs are set in **screen.json** files. First-level tabs are in **menu** block. It can have only a view or some child-views. For example:
+```ts
+{
+  "name": "example",
+  "title": "Example Title",
+  "primaryViewName": "no-child-view",
+  "primaryViews": [
+    "no-child-view"
+  ],
+
+  "_comment": "Primary view is a view set by default;"
+
+  "navigation": {
+    "menu": [
+      {
+        "viewName": "no-child-view"
+      },
+      {
+        "title": "View with children",
+        "child": [
+
+        "_comment": "Here comes second-level tabs"
```

Comment:
Not a valid json (`child` is an array) and I'm not sure if unrecognized properties are allowed.

It probably would be better to comment ts-style 

```ts
// "Here comes second-level tabs
"child": [
  { "viewName": "first-child" },
  { "viewName": "second-child" }
  // children can also have their own children, making third-level menu and so on
]
```